x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlExecutionInfo.java

```diff
@@ -70,6 +72,7 @@ public class EsqlExecutionInfo implements ChunkedToXContentObject, Writeable {
     private final transient Predicate<String> skipUnavailablePredicate;
     private final transient Long relativeStartNanos;  // start time for an ESQL query for calculating took times
     private transient TimeValue planningTookTime;  // time elapsed since start of query to calling ComputeService.execute
+    private boolean isPartial; // Does this request have partial results?
```

Comment:
It's not clear to me we really need this field. If we can instead rely on the looking looking for SKIPPED or PARTIAL status in each cluster we'll know it's partial without having to add this and a new transport version.